### PR TITLE
update Jenkinsfile to enable specifying DOOZER_DATA_PATH

### PR DESCRIPTION
**Pull request by shiywang**
…DATA_PATH

### CODE REVIEW

**Comment by tbielawa** [comment]
I am very curious as to why you would want this.

**Comment by shiywang** [comment]
@tbielawa  the use case is when I testing RHEL8 images in the future, we haven't had openshift-4.2-rhel8 branch something like this, so we might need to use a personal repo's branch.

the parameter already picked up in here https://localhost:8888/job/hack/job/shiywang-aos-cd-jobs/job/build%252Fcustom/10/console 

@tbielawa @sosiouxme PTAL

File: jobs/build/custom/Jenkinsfile [hunk 1]
```diff
@@ -113,7 +129,7 @@ node {
                 if (rpms.toUpperCase() != "NONE") {
                     currentBuild.displayName += rpms.contains(",") ? " [RPMs]" : " [${rpms} RPM]"
                     currentBuild.description = "building RPM(s): ${rpms}\n"
-                    command = "--working-dir ${doozer_working} --group 'openshift-${params.BUILD_VERSION}' "
+                    command = "--working-dir ${doozer_working} --data-path ${doozer_data_path} --group 'openshift-${params.BUILD_VERSION}' "
```
Comment (sosiouxme) [inline jobs/build/custom/Jenkinsfile]: hmm, I realize it was this way when you found it, but the longer it gets, the more obvious... why isn't this just:
```suggestion
                    command = doozerOpts
```

and the same for the next couple changes

**Comment by shiywang** [comment]
@sosiouxme updated, PTAL

**Pull request merged**